### DISCUSSION TITLE
Custom Color Saver

### DISCUSSION THREAD

**Discussion by onehivesystems** [grafana/grafana#49356] [Dashboards, Panels & Visualizations]
NO LONGER NEEDED AS THIS WAS CONVERTED INTO A UI REQUEST

Been using Grafana for a few years now. I have some specific colors that pop well on monitors for status dashboards, but I have to store the RGB values in a text document, then copy and paste them in the custom selector each time I want to use them. I was wondering if it would be possible to have some kind of custom color storage when the color selector appears.

Since the color selector already has the tabs "Colors" and "Custom": maybe there could be a third tab like "Stored" that has all the colors a user creates stored in it with maybe an add button to create a new color.

![Screenshot_20220521-000536_Chrome](https://user-images.githubusercontent.com/77515137/169634828-df602442-286e-4b4d-ad70-8d39eb93fb86.jpg)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
